spec/features/book_page_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Books' do
+  let(:book_page) { BookPage.new }
+  let(:count_books) { stub_const 'Constants::BOOKS_ON_PAGE', 12 }
```

Comment:
Use value less than 12
You don't need to create 12 books to testing pagination
`2` will be enough